docs/setup-guide/run-e2e-test-locally.md

```diff
@@ -0,0 +1,68 @@
+# How to run e2e test in local dev environment
+
+This doc is to describe how to run e2e test in your local dev environment. Please note, local dev environment is not fitting for many existing e2e test cases. So if those tests failed, it's not a surprise.
+
```

Comment:
Add a section Prerequisite, let dev knows if they following instruction on https://github.com/futurewei-cloud/arktos/blob/e60641d5ee8196f284583fbd0473be3efa383e08/docs/setup-guide/setup-dev-env.md to set up dev environmet, they will need to install docker before running kubetest

Reply:
Please update setup-dev-env.md. Add the following section from your script:
1. Install docker
2. Install crictl (replace section "Install Containerd")

Reply:
Added